frontstage/views/register.py

```diff
@@ -27,16 +27,17 @@ def validate_enrolment_code(enrolment_code):
     # Build the URL
     # url = app.config['API_GATEWAY_IAC_URL'] + '{}'.format(enrolment_code)
     url = app.config['RM_IAC_GET'].format(app.config['RM_IAC_SERVICE'], enrolment_code)
-    logger.debug('Validate IAC URL is: {}'.format(url))
+    logger.debug('"event" : "Logging IAC URL", "URL": "{}"'.format(url))
```

Comment:
I think splunk will be fine with this, though to be exact I think it's better to be `event: "Logging IAC URL"` without the space before : or the quotes around event. We'll have to ask splunk guys about exact formatting